pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -79,7 +80,7 @@ func (c *ConnectionRecorder) SaveInvitation(invitation *Invitation) error {
 		return err
 	}
 
-	return c.store.Put(k, bytes)
+	return c.transientStore.Put(k, bytes)
```

Comment:
I wonder if an invitation is transient...

Reply:
I agree will change to permanent store